scripts/visualise_results.py

```diff
@@ -34,16 +35,21 @@ def plot_data(df, number_of_days_to_plot):
 
     folders = sorted(glob(f"{parsed_variables['output_dir']}/*"))
     results = get_results(folders, number_of_days)
+
     result_fields = [
         [
             result[0],
-            result[1].average_weighted_get,
-            result[1].average_weighted_post,
-            result[1].average_weighted_total,
+            *(
+                result[1].percentiles[percentile]
+                for percentile in BenchmarkStats.PERCENTILES_TO_GRAPH
+            ),
```

Comment:
`results` is a `NamedTuple` with `date` and `statistics` fields, so `result[0]` can be replaced with `result.date` and `result[1]` with `result.statistics`.

Reply:
Good shout!. Updated.